expando.user.js

```diff
@@ -43,54 +43,53 @@
             }
             req.setRequestHeader('X-Requested-With', 'XMLHttpRequest');
 
-            req.send(serialize(data));
+            req.send((function serialize(obj) {
```

Comment:
You've inlined the function here, but didn't get rid of the original 20 or so lines up.

Reply:
Other than that, good to go :)